### DISCUSSION TITLE
"Overtake Camera/View"

### DISCUSSION THREAD

**Discussion by kicker414** [f1multiviewer/issue-tracker#500] [Feature requests]
A custom "overtake camera/view" of sorts which is a view that automatically switches to the driver camera of a driver who might be in an overtaking position. The simplest integration I could think of would be to look at driver intervals and based on a custom criteria, switch to that drivers POV camera. For example, switch to the driver camera if driver interval is less than or equal to 0.200 seconds.

Additional settings could include: 
Setting minimum hold time (i.e. it won't switch to another camera unless 3 seconds have passed, to prevent rapid switching)
Driver priority (i.e. set drivers who you want to see over others if they both meet the same criteria)
DRS detection (i.e. either prioritize drivers with DRS active or increase the interval for drivers with DRS active)

During the start and restarts/SC's I could see this being not very useful, but it would be nice to have the camera automatically switching during the race to see overtakes or close battles that aren't broadcast and if you aren't paying attention to the whole field.

**Comment by jeppeolesen** [comment]
Redirecting this to #68
